src/sap/fhir/model/r4/lib/FHIRBundleEntryFullUrlType.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * ${copyright}
+ */
+
+// Provides enumeration sap.fhir.model.r4.FHIRBundleEntryFullUrlType
+sap.ui.define(function() {
+	"use strict";
+
+	/**
+	 * Types to generate the fullUrl in FHIRBundleEntry {@link sap.fhir.model.r4.lib.FHIRBundleEntry}
+	 *
+	 * @alias sap.fhir.model.r4.lib.FHIRBundleEntryFullUrlType
+	 * @author SAP SE
+	 * @enum {string}
+	 * @public
+	 * @since 1.0.0
+	 * @version ${version}
+	 */
+	var FHIRBundleEntryFullUrlType = {
```

Comment:
rename file to `URI.js` and place it under `sap/fhir/model/r4/type`

Reply:
also  transform this enum to a child of `sap/ui/model/SimpleType`

Reply:
done